go/service/user.go

```diff
@@ -737,10 +737,27 @@ func (h *UserHandler) ReportUser(ctx context.Context, arg keybase1.ReportUserArg
 		arg.Username, arg.IncludeTranscript, convIDStr),
 		func() error { return err })()
 
-	if arg.IncludeTranscript && arg.ConvID == nil {
-		return errors.New("invalid arguments: IncludeTranscript is true but ConvID == nil")
+	if arg.IncludeTranscript {
+		if arg.ConvID == nil {
+			return errors.New("invalid arguments: IncludeTranscript is true but ConvID == nil")
+		}
+		return errors.New("`IncludeTranscript` is not implemented")
```

Comment:
so this is always going to be false for the release?

Reply:
hopefully... this PR also comments the checkbox out in the GUI

Reply:
maybe safer to log and ignore for now if it's not clear. so we don't end up with a release where blocking is impossible in some cases

Reply:
good idea